waspstation/code/modules/mining/deepcore/ore_vein.dm

```diff
@@ -0,0 +1,37 @@
+GLOBAL_LIST_EMPTY(ore_vein_landmarks)
+
+/obj/effect/landmark/ore_vein
```

Comment:
In the event we want to create ruins with specific ore veins, subclass this with one of each ore type.